### PR TITLE
Refactor: small refactor on workflow-definition-service

### PR DESCRIPTION
**Pull request by eloycoto**

### CODE REVIEW

File: workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java [hunk 1]
```diff
@@ -106,30 +106,26 @@ public WorkFlowDefinitionResponseDTO save(String workFlowName, WorkFlowType work
...
 		}
+
+		workFlowDefinition.setType(workFlowType);
+		workFlowDefinition.setParameters(stringifyParameters);
+		workFlowDefinition.setModifyDate(new Date());
+		workFlowDefinition.setProperties(propertiesDefinition);
+		workFlowDefinition.setProcessingType(workFlowProcessingType.name());
+		workFlowDefinition.setNumberOfWorks(works.size());
+
+		workFlowDefinition = workFlowDefinitionRepository.save(workFlowDefinition);
```
Comment (RichardW98) [inline workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java:127]: we only want to save the workflowDefinition if it's not found in db or updated.

Comment (eloycoto) [reply workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java:127]: The thing is, does it matter in terms of complexity? At the moment, if the works change inside, but keep the same number will never be updated. 

Maybe, what we can do, it's avoid that save() and only use the last one (finally) something that I'm happy to do

Comment (RichardW98) [reply workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java:127]: no I think this is good,   let's update it every time when app restarts

**Review by RichardW98 — APPROVED** [review]

**Pull request merged**